controllers/case-controller.js

```diff
@@ -97,6 +97,7 @@ const CaseController = (Case, sequelize) => {
         dateTo: body.dateTo || null,
         phoneNumber: body.phoneNumber ? `%${body.phoneNumber.replace(notDigits, '')}%` : null,
         counselor: body.counselor || null,
+        closedCases: typeof body.closedCases === 'undefined' ? true : body.closedCases,
```

Comment:
Here you can use this comparison to avoid the ternary (but please double check it ;) )
`typeof body.closedCases === 'undefined' || body.closedCases`